bftengine/src/communication/PlainTcpCommunication.cpp

```diff
@@ -687,25 +759,28 @@ class PlainTCPCommunication::PlainTcpImpl {
                uint32_t bufferLength,
                uint16_t listenPort,
                uint32_t maxServerId,
-               string listenIp) :
+               string listenIp,
+               UPDATE_CONNECTIVITY_FN statusCallback) :
       _selfId{selfNodeId},
       _listenPort{listenPort},
       _listenIp{listenIp},
       _bufferLength{bufferLength},
       _maxServerId{maxServerId},
-      _nodes{nodes} {
-
-    if (_selfId < _maxServerId) {
+      _statusCallback{statusCallback} {
+    // all replicas are in listen mode
+    if (_selfId <= _maxServerId) {
```

Comment:
Was this the multi-client bug you fixed? Whether it was or not, could you please add some description to the PR about what the bug was, and how you fixed it? "Fixed a bug" doesn't tell the next maintainer much.

Reply:
I changed the title and the description to me more precise

Reply:
Looks good. Thanks for the update!